do-not-commit/http-not-https-connection.py

```diff
@@ -0,0 +1,7 @@
+import urllib3 as ur3
+
+# ruleid:http-not-https-connection
+pool = ur3.connectionpool.HTTPConnectionPool("example.com")
```

Comment:
Detected HTTPConnectionPool. This will transmit data in cleartext.
It is recommended to use HTTPSConnectionPool instead for to encrypt
communications. ([dit.network.http-not-https-connection.http-not-https-connection](https://semgrep.dev/r/dit.network.http-not-https-connection.http-not-https-connection))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=dit.network.http-not-https-connection.http-not-https-connection) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>